package/yast2-metapackage-handler.spec

```diff
@@ -17,38 +17,34 @@
 
 
 Name:           yast2-metapackage-handler
-Version:        4.3.0
+Version:        4.3.1
```

Comment:
as it is for master and differ from 15.3 please bump to 4.4.0

Reply:
done